lxc/exec.go

```diff
@@ -20,10 +22,29 @@ func (c *execCmd) usage() string {
 	return gettext.Gettext(
 		"Execute the specified command in a container.\n" +
 			"\n" +
-			"lxc exec container [command]\n")
+			"lxc exec container [--env EDITOR=/usr/bin/vim]* [command]\n")
```

Comment:
On Thu, Feb 26, 2015 at 04:13:11PM -0800, St�phane Graber wrote:
> > @@ -20,10 +22,29 @@ func (c *execCmd) usage() string {
> >  	return gettext.Gettext(
> >  		"Execute the specified command in a container.\n" +
> >  			"\n" +
> > -			"lxc exec container [command]\n")
> > +			"lxc exec container [--env EDITOR=/usr/bin/vim]* [command]\n")
> 
> Is that the standard syntax? I've always used:
> 
>     lxc exec container [--env EDITOR=abc]... [command]
> 
> To denote that an option may be used multiple times.

I don't know, I just made it up. I can change it.

> Also, I believe the command is required nowadays, so we should make that \<command\>
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/281/files#r25478769